### PR TITLE
Update nuget push

### PR DESCRIPTION
**Pull request by JosephDSchwartz**
# Summary of PR changes

## PR Requirements
- [x] For major, minor, or breaking changes, at least one of the commit messages contains the appropriate `+semver:` keywords.
  - See the *Incrementing the Version* section of the repository's README.md for more details.
- [x] The action code does not contain sensitive information.

*NOTE: If the repo's workflow could not automatically update the `README.md`, it should be updated manually with the next version.  For javascript actions, if the repo's workflow could not automatically recompile the action it should also be updated manually as part of the PR.*

### CODE REVIEW

File: .build-linux/publishViewsInSchema.ps1 [hunk 1]
```diff
@@ -132,8 +131,9 @@ ForEach-Object {
     $nuspec = '<?xml version="1.0"?><package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd"><metadata>'
     $nuspec += "<id>$packageId</id>"
     $nuspec += "<version>$version$metadata</version>"
-    $nuspec += "<authors>$machineName</authors>"
+    $nuspec += "<authors>$packageId</authors>"
     $nuspec += "<description>Table version of the $packageId view.</description>"
+    $nuspec += "<repository url=`"$RepositoryUrl`"></repository>"
```
Comment (danielle-casella-adams) [inline .build-linux/publishViewsInSchema.ps1:136]: I asked this on your other PR but will this behave if the repository url is empty?

File: .build-linux/publishViewsInSchema.ps1 [hunk 2]
```diff
@@ -132,8 +131,9 @@ ForEach-Object {
     $nuspec = '<?xml version="1.0"?><package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd"><metadata>'
     $nuspec += "<id>$packageId</id>"
     $nuspec += "<version>$version$metadata</version>"
-    $nuspec += "<authors>$machineName</authors>"
+    $nuspec += "<authors>$packageId</authors>"
```
Comment (danielle-casella-adams) [inline .build-linux/publishViewsInSchema.ps1:134]: Why this change?

Comment (JosephDSchwartz) [reply .build-linux/publishViewsInSchema.ps1:134]: The machine name makes little sense when we're working with action runners. Honestly it didn't make much sense when we were working with other build machines. The `packageId` seemed like a good enough replacement 🤷‍♂️ Maybe we should have a parameter to customize it, but I can always come back and do something like that if we really want.

File: .build-linux/publishViewsInSchema.ps1 (hunk 1 above)
Comment (JosephDSchwartz) [reply .build-linux/publishViewsInSchema.ps1:136]: I'll answer here too for posterity:

Nothing seems to happen other than the package isn't associated with a repository. If you enter a bad url the same thing happens. I did notice some odd behavior in my tests that makes me think if packages of the same name already exist and you mess up the repository url then the package still acts like the previous packages before it (i.e. will be associated with a repository or not depending on whether previous packages were). I assume that's some GitHub black magic behind the scenes.

**Comment by bbelnap** [comment]
Is this PR going to be merged soon? I want to be able to reference the new version in my workflow.

**Review by hpractv — COMMENTED** [review]
👍🏽

**Review by hpractv — APPROVED** [review]
👍🏽 x 2
